Resources/public/js/views/services/ez-universaldiscoveryviewservice.js

```diff
@@ -22,6 +22,51 @@ YUI.add('ez-universaldiscoveryviewservice', function (Y) {
      */
     Y.eZ.UniversalDiscoveryViewService = Y.Base.create('universalDiscoveryViewService', Y.eZ.ViewService, [Y.eZ.SideViewService], {
         /**
+         *
+         * @method _load
+         * @protected
+         * @param {Function} callback
+         */
+        _load: function (callback) {
+            var startingLocation = new Y.eZ.Location(),
+                path = [];
+
+            if (this.get('parameters').startingLocationId) {
+                startingLocation.set('id', this.get('parameters').startingLocationId);
+                this._loadStartingLocationPath(startingLocation, Y.bind(function(locationPath) {
+                    startingLocation.get('path').unshift(this.get('virtualRootLocation'));
```

Comment:
you can not do that. It seems you also need in addition the virtual root, so you should make sure it's passed to the UDW view alongside the startingLocation